packages/rum-core/src/domain/contexts/ciTestContext.ts

```diff
@@ -5,6 +5,10 @@ export interface CiTestWindow extends Window {
 }
 
 export function getCiTestContext() {
+  if (window['dd_ci_visibility_test_context']) {
+    return window['dd_ci_visibility_test_context'];
```

Comment:
what we include here is what we end up using in the UI to query for RUM sessions, so I think it'd be better to be explicit about what we're including. Something like:

```javascript 
if (window['dd_ci_visibility_test_context']) {
  return {
    test_execution_id: window['dd_ci_visibility_test_context'].testExecutionId,
  }
}
```

this means that we have a hard contract between selenium and RUM but I think that's actually a good thing 😄

Reply:
My intention was to let us change the context without having to modify this function in the future.
Although it's not a big deal, changes as per your suggestion